nixos/doc/manual/development/option-def.xml

```diff
@@ -74,6 +74,12 @@ services.openssh.enable = mkOverride 10 false;
    discarded. The function <varname>mkForce</varname> is equal to
    <varname>mkOverride 50</varname>.
   </para>
+  <para>
+   The function <varname>mkOverrideUnset</varname> can be used instead to
+   remove an attribute that was defined in another module. It also takes a
+   priority, but no replacement value. <varname>mkUnset</varname> is equal to
```

Comment:
Generally I would avoid starting a sentence with a piece of code. Perhaps something like
```suggestion
   priority, but no replacement value. For convenience, <varname>mkUnset</varname> is equal to
```
?

Reply:
Sure, done!